app/templates/PACKAGE/_SCRIPTS.json

```diff
@@ -1,9 +1,10 @@
 {
     "scripts": {
-        "gulp": "gulp --require @babel/register",
-        "build": "cross-env NODE_ENV=production npm run gulp",
-        "lint": "npm run gulp scripts:lint styles:lint",
-        "watch": "npm run gulp watch",
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "build:legacy": "cross-env BROWSERSLIST_ENV=legacy webpack",
```

Comment:
Split code for modern and legacy browsers is a great thing but it adds some inconveniences to the developing process:
1. The decision which bundle to use should be made on the backend, probably by checking the user agent. I'm not sure if each project is ready to implement this extra logic. And I guess we currently don't have a solution from the box.
2. If we use `legacy` polyfilled code by default during the development, we need to check the `modern` code as well in order to make sure it works the same way. That's kinda extra work :)
3. Polyfilled files are outputting to the folder named `legacy`, which is a common way to name bundle for old browsers, but also might be confused with "legacy" code in the project if it has some :)

Probably we'd better make a code-splitting optional if it is possible, what do you think? :)

Reply:
@sme12 yeah, I also been thinking whether we need to have this sort of setup by default... 

1. There's also this `<script nomodule>` approach. It requires a bit of JS for browsers that don't understand `nomodule` attribute. Frankly, I need to revisit this topic to see if it makes more sense.
2. That's true. Even though one could argue that you mostly do development in your favorite browser and then still have to test in, say, IE afterwards. But yes, it adds up this extra step of `npm run build`. In a traditional setup you can go away with the results yielded by already running `npm start`.
3. `legacy`/`modern` words in context of bundling are somewhat common, for example [here](https://github.com/philipwalton/webpack-esnext-boilerplate/blob/master/tasks/bundles.js#L79-L123) and [here](https://github.com/babel/preset-modules#installation--usage), so I just excused myself from another naming task 😁 However, I see your concern, among other reasonable choices I found were `classic`/`module` as seen [here](https://github.com/philipwalton/blog/blob/master/tasks/javascript.js#L108-L197) (although they convey a slightly different meaning compared to the current solution) and simply `old`/`new` (I also like that both words have equal number of characters 🙂).